src/_test_/movieCounter.test.js

```diff
@@ -0,0 +1,23 @@
+/** @jest-environment jsdom */
```

Comment:
Good job adding test for the movieCounter. After fixing your movieCounter function, you need to fix your tests here and add a mock environment. You can use follow the example I shared for the commentsCounter. 👍